### PR TITLE
Create iPayments.js

### PR DESCRIPTION
**Pull request by TricksXTech**
Add Now

### CODE REVIEW

File: iPayments.js [hunk 1]
```diff
@@ -0,0 +1,145 @@
+let libPrefix = "iPayments" 
+var price_api_url = "https://api.i-payments.site/check/price"
```
Comment (mpz) [inline iPayments.js:2]: var api_url = "https://api.i-payments.site/"

use this only.
Then you can use later:
api_url + "/check/price" and etc

File: iPayments.js [hunk 2]
```diff
@@ -0,0 +1,145 @@
+let libPrefix = "iPayments" 
+var price_api_url = "https://api.i-payments.site/check/price" 
+var build_api_url = "https://api.i-payments.site/v2/build" 
+var balance_api_url = "https://api.i-payments.site/v2/balance" 
+var deposit_api_url = "https://api.i-payments.site/v2/deposit" 
+var transfer_api_url = "https://api.i-payments.site/v2/transfer" 
+ 
+function setBashKey(key){ 
+Bot.setProperty(libPrefix + "bashkey", key, "string");
```
Comment (mpz) [inline iPayments.js:9]: no formatting

File: iPayments.js [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+var balance_api_url = "https://api.i-payments.site/v2/balance" 
+var deposit_api_url = "https://api.i-payments.site/v2/deposit" 
+var transfer_api_url = "https://api.i-payments.site/v2/transfer" 
+ 
+function setBashKey(key){ 
+Bot.setProperty(libPrefix + "bashkey", key, "string"); 
+} 
+ 
+function setPrivateKey(key){ 
+Bot.setProperty(libPrefix + "privatekey", key, "string");
```
Comment (mpz) [inline iPayments.js:13]: no formatting

File: iPayments.js [hunk 4]
```diff
@@ -0,0 +1,145 @@
...
+Bot.sendMessage('Use : `Libs.iPayments.checkBalance("TRX");`'); 
+ return 
+} 
+HTTP.get({ 
+  url:""+balance_api_url+"/?PrivateKey="+prikey+"&PublicKey="+pubkey+"&Currency="+currency+"", 
+  success: libPrefix + 'Balancee'   
+}) 
+} 
+ 
+function Balancee(){
```
Comment (mpz) [inline iPayments.js:88]: bad naming


need use small case in method

like

File: iPayments.js [hunk 5]
```diff
@@ -0,0 +1,145 @@
...
+function Generatee(){ 
+   return Bot.sendMessage(""+content); 
+} 
+ 
+function checkBalance(currency){ 
+  let devkeys = loadKey(); 
+  let prikey = devkeys.privateKey 
+  let pubkey = devkeys.publicKey 
+ if(!currency){ 
+Bot.sendMessage('Use : `Libs.iPayments.checkBalance("TRX");`');
```
Comment (mpz) [inline iPayments.js:79]: don't use Bot.sendMessage in the lib

**Review by mpz — CHANGES_REQUESTED** [review]

File: iPayments.js (hunk 1 above)
Comment (TricksXTech) [reply iPayments.js:2]: Using this i got 0 error

File: iPayments.js (hunk 2 above)
Comment (TricksXTech) [reply iPayments.js:9]: It's Current Bot's set Property
And Check Coinpayments Lib too

File: iPayments.js (hunk 3 above)
Comment (TricksXTech) [reply iPayments.js:13]: Same reply as first

File: iPayments.js (hunk 4 above)
Comment (TricksXTech) [reply iPayments.js:88]: Uppercase Letter And Lowercase Have There Own existence
Where as most of the bug got with lowercase

File: iPayments.js (hunk 5 above)
Comment (TricksXTech) [reply iPayments.js:79]: You make it correct

**Pull request closed without merging**